server/inspect.go

```diff
@@ -16,11 +16,12 @@ import (
 )
 
 func (s *Server) getIDMappingsInfo() types.IDMappings {
+	MaxInt32 := int(^uint32(0) >> 1)
```

Comment:
Docs for `math.MaxInt32 are [here][1].

[1]: https://golang.org/pkg/math/#pkg-constants

Reply:
I think we don't want to change the limit using MaxInt32, when we are on a 64-bits system, it should be like:

`MaxInt := int64(^uint(0) >> 1)`

Reply:
we probably want something like:

```
	max := int64(int(^uint(0) >> 1))
	if max > math.MaxUint32 {
		max = math.MaxUint32
	}
```

Reply:
This does not work. Since max ends up being a int64 and I can't assign to a int field.

../go/src/github.com/kubernetes-incubator/cri-o/server/inspect.go:29:8: cannot use max (type int64) as type int in field value

Why do we use the int64() wrapper?

Reply:
if we cast it to `int32` then it will get the lower value also on platforms where an int is 8 bytes.

We need to add a cast back to int before using max